Objects/complexobject.c

```diff
@@ -941,6 +941,17 @@ actual_complex_new(PyTypeObject *type, PyObject *args, PyObject *kwargs)
     if (PyUnicode_Check(arg)) {
         return complex_subtype_from_string(type, arg);
     }
+    else if (PyByteArray_Check(arg) || PyBytes_Check(arg)) {
+        if (PyByteArray_Check(arg)) {
+            const char *string = PyByteArray_AS_STRING(arg);
```

Comment:
Pls, note that there are multiple warnings

Reply:
Yeah, I learned a polite way to say "this code so broken, that will compile just on gcc".